images/mod-tile/start.py

```diff
@@ -185,7 +187,7 @@ def render_expired(currently_expired_tiles_file_path):
     """
     # TODO: consider extracting map, min-zoom and touch-from env var
     run_subprocess_async(
-        fr'cat {currently_expired_tiles_file_path} | /src/mod_tile/render_expired --map=osm --min-zoom=8 --touch-from=8 >/dev/null', True)
+        fr'cat {currently_expired_tiles_file_path} | /src/mod_tile/render_expired --map=osm --min-zoom=8 --touch-from=8 >/dev/null', log.info, log.error, True)
```

Comment:
I think zoom=14 is a bit performance heavy, what about 14 and above?

overall it should be configurable via env var

Reply:
created `TILE_EXPIRE_MIN_ZOOM` env var with a default value of 14. done